applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -521,70 +457,23 @@ static int change_active_audio_stream(void)
 	return 0;
 }
 
-static int change_active_brdcast_src(void)
-{
-	int ret;
-
-	if (active_stream.stream != NULL && active_stream.stream->ep != NULL) {
-		if (active_stream.stream->ep->status.state == BT_BAP_EP_STATE_STREAMING) {
-			ret = bt_bap_broadcast_sink_stop(broadcast_sink);
-			if (ret) {
-				LOG_ERR("Failed to stop broadcast sink: %d", ret);
-				return ret;
-			}
-		}
-	}
-
-	ret = bis_headset_cleanup();
-	if (ret) {
-		LOG_ERR("Error cleaning up");
-		return ret;
-	}
-
-	/* Wrap broadcaster names */
-	if (++active_stream.brdcast_src_name_idx >= ARRAY_SIZE(brdcast_src_names)) {
-		active_stream.brdcast_src_name_idx = 0;
-	}
-	LOG_INF("Switching to %s", brdcast_src_names[active_stream.brdcast_src_name_idx]);
-
-	LOG_DBG("Restarting scanning for broadcast sources");
-	ret = bt_bap_broadcast_sink_scan_start(NRF5340_AUDIO_GATEWAY_SCAN_PARAMS);
-	if (ret && ret != -EALREADY) {
-		LOG_ERR("Unable to start scanning for broadcast sources");
-		return ret;
-	}
-
-	return 0;
-}
-
 int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 {
-	int ret;
+	int ret = 0;
 
 	if (action == LE_AUDIO_USER_DEFINED_ACTION_1) {
 		LOG_DBG("User defined action 1");
 		ret = change_active_audio_stream();
 	} else if (action == LE_AUDIO_USER_DEFINED_ACTION_2) {
 		LOG_DBG("User defined action 2");
-		ret = change_active_brdcast_src();
 	} else {
```

Comment:
So if `action == LE_AUDIO_USER_DEFINED_ACTION_2` then `ret = 0`? Is that intentional?